.github/workflows/onesky.yml

```diff
@@ -0,0 +1,46 @@
+name: OneSky Translation PR
+
+on:
+  schedule:
+    # Run every day at 10:50am UTC (5:10am EST). To avoid delays and possibly dropped jobs GitHub
+    # recommends not scheduling jobs at the top of the hour.
+    - cron: '50 10 * * *'
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: write
+  pull-requests: write
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .tool-versions
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache
+      - name: 🌐 Extract Translations
+        run: yarn extract
+      - name: ⛅🔽 OneSky Download
+        env:
+          ONESKY_API_KEY: ${{ secrets.ONESKY_API_KEY }}
+          ONESKY_API_SECRET: ${{ secrets.ONESKY_API_SECRET }}
+          ONESKY_PROJECT_ID: ${{ secrets.ONESKY_PROJECT_ID }}
+        run: yarn onesky:download
+      - name: 🔀 Create PR
+        uses: peter-evans/create-pull-request@v7
+        env:
+          # Disable git hooks because we are only modifying translation files
+          HUSKY: 0
+        with:
```

Comment:
Do we want to add reviewers to this?

Reply:
What are your thoughts about adding `delete-branch: true` to delete thre branch after each merge and so the branch is only around if changes are to be made.

Reply:
> What are your thoughts about adding `delete-branch: true` to delete thre branch after each merge and so the branch is only around if changes are to be made.

We have this repo configured to automatically delete branches from merged PRs, so that shouldn't be an issue

Reply:
> Do we want to add reviewers to this?

Adding team reviewers requires a Personal Access Token with `repo` scope or making a GitHub App with those permissions (https://github.com/peter-evans/create-pull-request/tree/v7/#action-inputs). If we use a PAT, we have to deal with rotating it when it expires and having to get someone else to create one if the person who created it leaves our GitHub org or the `web-engineering-js` team. It's also less secure to use a long-lived PAT instead of the short-lived automatic `GITHUB_TOKEN`. Making a GitHub App for this also doesn't feel worth it.

If the goal is to be alerted when this PR gets created, I think we can mention `@CruGlobal/web-engineering-js` in the PR body, and we'll all get a notification.